client/src/components/Persons.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Modal from 'react-modal';
+
+import PersonsContent from './PersonsContent.jsx';
+
+Modal.setAppElement(document.getElementById('workspaces'));
+ 
+class Persons extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      show: false,
+      personsCount: 0
+    };
+  }
+
+  handleOpen() {
+    this.setState({ show: true });
+  }
+
+  handleClose() {
+    this.setState({ show: false });
+  }
+   
+  setPersons(personsCount) {
+    if (!isNaN(personsCount)) {
+      this.setState({ personsCount, show: false });
+    }
+  }
+
+  clearPersons() {
+    this.setPersons(0);
+  }
```

Comment:
Not sure I see the point of this extra method, could just call `setPersons` with a value of `0` from where it is invoked.